client/cli/src/lib.rs

```diff
@@ -931,6 +931,28 @@ where
 	Ok(config)
 }
 
+fn interface_str(
+	is_external: bool,
+	is_unsafe_external: bool,
+	is_validator: bool
+) -> Result<&'static str, error::Error> {
+	if is_external && is_validator {
```

Comment:
```suggestion
	if is_validator && !is_unsafe_external {
```

When you run a dev chain you probably don't care about the risk at all, it should be allowed to use `unsafe` flag to override this.